manifests/w/WHONET/AMRInterpretationEngine/23.6.21/WHONET.AMRInterpretationEngine.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WHONET.AMRInterpretationEngine
+PackageVersion: 23.6.21
+InstallerLocale: en-US
+InstallerType: wix
+Dependencies:
+  WindowsFeatures:
+  - Microsoft.DotNet.DesktopRuntime.6

```

```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```
I'm not familiar with the `WindowsFeatures` property. To my knowledge, only `PackageDependencies` is currently functional.